HISTORY.md

```diff
@@ -1,3 +1,11 @@
+1.15.3 / 2016-12-xx
```

Comment:
This should just be "unreleased" you can use past commits to see how that is done until we have docs on the process.